tests/test_data/fitslowcookerqueen.com/fitslowcookerqueen_2.json

```diff
@@ -59,7 +59,7 @@
   "total_time": 190,
   "cook_time": 180,
   "prep_time": 10,
-  "ratings": 5,
+  "ratings": 5.0,
```

Comment:
@jknndy something a bit odd going on here - a `.0` has been reintroduced to this test data?

Reply:
For ratings I think the .0 is the intended behavior, I ran a search and it seems shows 116 ratings keys have a .0.

Originally was thinking that this may be the right format as it makes it easier to differentiate between the `ratings` and `ratings_count` but after some thought I'm going to clean it up to keep the format consistent throughout the keys.

Reply:
Ah yep, my mistake - it was `ratings_count` I was confusing this with.  All good :)

Reply:
Ah, I misunderstood: when you said consistent, I thought you meant that all `ratings` keys should have single-decimal-place values -- could/should we revert 18c90dd5225666b9472a1793d6b6312d0a60e23f?

I think that'd make diffs easier to review when a rating changes from `5` to `4.93` (because it'll be clearer to review `5.0` to `4.9` than `5` to `4.9`)